2016-07-14-new-news.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: peter wins dublin 5000m track title 
+Location: irishtown stadium
+---
+
+Peter Arthur continued his fine track form when winning  the 
+Dublin Senior 5000 Metres at last nights Graded Meeting
+at Irishtown Stadium.
+
+Peter also recorded a new Personal Best over the distance
+with a finishing time of 15mins 5 seconds.After an opening lap
+of 73 secs a small group of six athletes including Peter went clear
+of the field and they passed 3000 Metres in 9 mins 10 secs.
+
+Peter was now in a group of three who were making the pace.
+With three laps to go he took control and immediately opened
+a gap on his two pursuers which he stretched as he went for the
+finish line. A last 200 metres of 33 seconds showed his superior
+speed and strength  was the  winning factor.
+
+Thi is Peters second senior title win this year as he also won the Leinster
```

Comment:
Note to self ..... Always spellcheck Jims race reports before trying to post.

Got ta go and do acupuncture treatment, so off air for a while..

Sent from my iPad

> On 15 Jul 2016, at 09:57, Mike Griffin <notifications@github.com> wrote:
> 
> In 2016-07-14-new-news.md:
> 
> > +Peter Arthur continued his fine track form when winning  the > 
> > +Dublin Senior 5000 Metres at last nights Graded Meeting
> > +at Irishtown Stadium.
> > +
> > +Peter also recorded a new Personal Best over the distance
> > +with a finishing time of 15mins 5 seconds.After an opening lap
> > +of 73 secs a small group of six athletes including Peter went clear
> > +of the field and they passed 3000 Metres in 9 mins 10 secs.
> > +
> > +Peter was now in a group of three who were making the pace.
> > +With three laps to go he took control and immediately opened
> > +a gap on his two pursuers which he stretched as he went for the
> > +finish line. A last 200 metres of 33 seconds showed his superior
> > +speed and strength  was the  winning factor.
> > +
> > +Thi is Peters second senior title win this year as he also won the Leinster
> "Thi" is missing an "s"
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>